docs/samples/explanation/aix/Fetch_20newsgroups/rfserver/rfserver/model.py

```diff
@@ -0,0 +1,58 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict
+import pickle
+import kserve
+
+
+class PipeStep(object):
+    """
+    Wrapper for turning functions into pipeline transforms (no-fitting)
+    """
+
+    def __init__(self, step_func):
+        self._step_func = step_func
+
+    def fit(self, *args):
+        return self
+
+    def transform(self, X):
+        return self._step_func(X)
+
+
+class RFModel(kserve.Model):  # pylint:disable=c-extension-no-member
```

Comment:
Any reason why we need to implement this? We can use the out of the box sklearn server directly to serve the model.

Reply:
@yuzisun sorry for letting you wait for so long
We are now using out of the box model 
thanks